internal/repo/mock/repo.go

```
@@ -199,6 +199,10 @@ func (r *InMemoryRepository) OffboardTenant(_ context.Context, schemaName string
 	return nil
 }
 
+func (r *InMemoryRepository) GetFilterOptions(ctx context.Context, resource repo.Resource, columns []repo.Filter, query repo.Query) error {
```

golangci-lint (2.11.4)

[error] The line is 139 characters long, which exceeds the maximum of 120 characters.

(lll)